src/coreComponents/unitTests/constitutiveTests/testTriaxial_druckerPragerExtended.xml

```diff
@@ -60,4 +60,12 @@
       nz="{ 1 }"
       cellBlockNames="{ cellBlock01 }"/>
   </Mesh>
+
+  <ElementRegions>
+    <CellElementRegion
+      name="dummy"
+      cellBlocks="{ cellBlock01 }"
+      materialList="{ dummy }"/>
+  </ElementRegions>
```

Comment:
Do we really want to initialize a `CellElementRegion` in the triaxial driver unit tests? I am not sure, but my impression was that we do not want to do that in this case?

Reply:
The problem here is that we do have an `InternalMesh` that builds one unique  `cellblock` named `cellBlock01`.
The new design break the _generator_ part (`InternalMesh` and `CellBlock*`) from the `ElementRegions` `CellElement(Sub)Region`. And it happens that `geosx` is not working without `ElementRegions`/`CellElementRegion`.

It happened that I had to force the creation.

But I kind of prefer having one dummy `CellElementRegion` receiving the result of a dummy (too) generator `InternalMesh`/`CellBlock` that are now deleted, than keeping the dummy generators in the air.

PS: look at line 50 and do not worry, somebody will resole this soon 😉 !